pkg/requests/base.go

```diff
@@ -173,8 +176,31 @@ func (rb *Base) InitFlags() {
 	rb.Cmd.Flags().MarkHidden("api-base") // #nosec G104
 }
 
+// MakeMultiPartRequest will make a multipart/form-data request to the Stripe API with the specific variables given to it
```

Comment:
```suggestion
// MakeMultiPartRequest will make a multipart/form-data request to the Stripe API with the specific variables given to it.
We need some comments to clarify how to use this method.
```suggestion
// Similar to making a multipart request using curl, add the local filepath to params arg with @ prefix.
// e.g. params.AppendData([]string{"photo=@/path/to/local/file.png"})
```
```